agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/mustering-game-state/player-mustering-game-state/PlayerMusteringGameState.ts

```diff
@@ -281,7 +281,9 @@ export default class PlayerMusteringGameState extends GameState<ParentGameState>
                 .filter(r => r.type.kind == RegionKind.SEA)
             )
             // Can't recruit in a enemy-controled territory
-            .filter(r => r.getController() == null || r.getController() == this.house);
+            .filter(r => r.getController() == null || r.getController() == this.house)
+            // Can't recruit into a blocked region, e.g. an adjacent blocked sea
+            .filter(r => r.garrison != 1000);
```

Comment:
Why not simply `r.garrison > 0`?

Reply:
Then you wouldnt be able to muster in your capital anymore :)

Reply:
Ah, true that. Maybe `r.garrison > 0 ? r.getController() == this.house : true`.

It's a small change, but it's to support possible neutral garrisons on seas. On top of that, I'd prefer to make the `1000` magic value as rare as possible in the codebase. 🙂

Reply:
Ok np. I will change that after I rebased the other PRs